.github/workflows/release-benchmarks.yml

```
@@ -39,7 +39,7 @@ jobs:
           cache: false
 
       - name: Set up just
-        uses: ./.github/actions/setup-just
+        uses: $/.github/actions/setup-just
```

actionlint (1.7.12)

[error] specifying action "$/.github/actions/setup-just" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)